Sources/GRPCCore/Documentation.docc/Tutorials/Hello-World/Hello-World.tutorial

```diff
@@ -76,12 +76,15 @@
   }
 
   @Section(title: "Update and run the application") {
-    You need to regenerate the stubs as the service definition has changed. You'll learn how to do
-    this in another tutorial.
+    You need to regenerate the stubs as the service definition has changed. To do this run the
+    following command from the root of the checked out repository:
 
-    To update the stubs run `Protos/generate.sh` from the root of the checked out repository. This
-    However, you still need to implement and call the new method in the human-written parts
-    of your application.
+    ```console
+    Protos/generate.sh
+    ```
+
+    You'll learn how to do this manually in another tutorial. Now that the stubs have been updated
```

Comment:
This is more of a mental note, but if we can link to other tutorials from here, it may be nice to do so.